frontend/src/pages/ActivityReport/Pages/Review/Submitter/NeedsAction.js

```diff
@@ -56,8 +62,9 @@ const NeedsAction = ({
           { approvingManager.name || 'No manager notes' }
         </div>
       </div>
+      {hasIncompletePages && <IncompletePages incompletePages={incompletePages} />}
       <div className="margin-top-3">
-        <Button disabled={!valid} onClick={submit}>Re-submit for Approval</Button>
+        <Button onClick={submit}>Re-submit for Approval</Button>
```

Comment:
I know we're showing the incomplete pages right above, but would it still make sense to have the button be disabled until the user has completed all the steps?

Reply:
I'd probably rather it not be disabled so it matches the rest of the form, but I could be convinced otherwise.

Reply:
I'm good with this implementation, since the Incomplete pages error info is directly above the button.